fix: Remove usages of devstack_docker.

dianakhuang opened the pull request:
The devstack_docker settings file was removed from the edx-platform repository, and we had a bunch of references to it, including in the provisioning scripts.

The correct way to handle this is to use the devstack.py file, which is mounted in the host already.

This also adds in some devstack-specific cms.yml and lms.yml configuration files that then get mounted into the lms and cms containers. These are ported from the devstack-experimental.yml files that were previously in edx-platform and were removed. https://github.com/openedx/edx-platform/pull/36908

----

I've completed each of the following or determined they are not applicable:

- [ ] Made a plan to communicate any major developer interface changes (or N/A)

clintonb commented:
I misread the switch. Why the change to from 18010? If you make this change here you will need to update the images and configuration. See https://github.com/edx/configuration/blob/master/docker/build/edxapp/Dockerfile#L34.

dianakhuang committed:
fix: Remove usages of devstack_docker.

The devstack_docker settings file was removed from the edx-platform
repository, and we had a bunch of references to it, including in the
provisioning scripts.

The correct way to handle this is to use the devstack.py file, which is
mounted in the host already.

dianakhuang committed:
fix: Add and mount lms/cms configuration files.

These were removed from the edx-platform repository.
Moving them here so devstack can still use them.
